### PR TITLE
Begin BBcode editor integration

### PR DESCRIPTION
**Pull request by beaver-dev**
More tags to come : large, small, colors…
Clearer assets folder structure (some assets have been moved)

### CODE REVIEW

File: style/FeatherBB/view/footer.php [hunk 1]
```diff
@@ -142,7 +142,7 @@
 
 </section>
 
-<script src="<?php echo get_base_url() ?>/style/FeatherBB/phone.min.js"></script>
+<script src="<?php echo get_base_url() ?>/js/phone.min.js"></script>
```
Comment (adaur) [inline style/FeatherBB/view/footer.php]: Why did you change that? This file only needs to be loaded if we have FeatherBB style on

**Comment by beaver-dev** [comment]
Because i get used to group all assets by type with other MVC frameworks : all javascripts files in the same directory, same for stylesheets.
But given the few number of files, that's of course not compulsory !

**Comment by adaur** [comment]
I'd prefer to keep it in the style folder for the moment if you don't mind

**Comment by beaver-dev** [comment]
File move back to styles :)

**Comment by adaur** [comment]
Excellent, thank you!

I really like what you did here. Two more things (that I can do if you're fed up :laughing:) : can you apply your changes to /view/viewtopic.php and /view/post.php as well, and change the function name from edToolbar to something more personal, such as display_toolbar?

**Comment by beaver-dev** [comment]
Thanks ! That can be done either tonight or tomorrow evening.
A few more BBcode tags should follow in days to come.

**Comment by beaver-dev** [comment]
Thanks !
That's done. I tried to add the "small" and "large" tags : created the buttons in toolbar and added some code in include/bbcd.php file, but changed are not visible on messages. As I don't know exactly how this bbcd file works it will be faster to ask you directly : do tags need to be enabled somewhere ? Could you link me some doc, or something like that ?

**Comment by adaur** [comment]
Thank you. A "id="req_message"" was missing in view/post.php, I have added it.

**Comment by adaur** [comment]
I almost forgot the edit.php file ! Will look into it tomorrow

**Comment by adaur** [comment]
https://github.com/featherbb/featherbb/commit/4b7f040775ad5278daa39add754b3d477b9a3177
